libsolidity/formal/SolverInterface.h

```diff
@@ -44,18 +44,18 @@ enum class CheckResult
 
 enum class Sort
```

Comment:
What does sort means here? Is it *kind* but the lingo in SMT is *sort*?

Reply:
I'm not sure, but z3 calls this "sort"

Reply:
Take a look here: https://github.com/ethereum/solidity/blob/develop/libsolidity/formal/Z3Interface.cpp#L181